models/repo/pushmirror.go

```diff
@@ -93,6 +94,14 @@ func GetPushMirrorsByRepoID(repoID int64) ([]*PushMirror, error) {
 	return mirrors, db.GetEngine(db.DefaultContext).Where("repo_id=?", repoID).Find(&mirrors)
 }
 
+// GetPushMirrorsByRepoIDWithSyncOnCommit returns push-mirrors for this repo that should be updated by new commits
+func GetPushMirrorsSyncedOnCommit(repoID int64) ([]*PushMirror, error) {
```

Comment:
The comment does not match the function name.

I think `GetPushMirrorsByRepoIDWithSyncOnCommit` is the better name or `GetSyncOnCommitPushMirrorsByID`

but if you've already had other reviews that suggested `GetPushMirrorsSyncedOnCommit` then simply:

```suggestion
// GetPushMirrorsSyncedOnCommit returns push-mirrors for this repo that should be updated by new commits
func GetPushMirrorsSyncedOnCommit(repoID int64) ([]*PushMirror, error) {
```